### DISCUSSION TITLE
Argo events Disaster Recovery

### DISCUSSION THREAD

**Discussion by naveenb30** [argoproj/argo-events#1702] [Q&A] [answered]
We are running argo events on EKS cluster, just want to check is there any way we can export argo-events sources and sensors config and import on a new EKS cluster.

For argo workflows including some of the cron jobs we are following this: https://argoproj.github.io/argo-workflows/disaster-recovery/

`kubectl get wf,cwf,cwft,wftmpl -o yaml > backup.yaml`

`kubectl apply -f backup.yaml`

Does argo events have anything similar ?

**Comment by whynowy** [comment, marked as answer]
That should be similar, 

`kubectl get es, sensor, eventbus -o yaml > xxx.yaml`

`kubectl apply -f xxx.yaml`

**Reply by naveenb30** [reply]
Great !!! Thanks much @whynowy
